content/pt-br/docs/tasks/configure-pod-container/configure-pod-initialization.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Configurando a Inicialização do Pod
+content_type: task
+weight: 130
+reviewers:
+---
+
+<!-- overview -->
+Esta página mostra como usar um contêiner de incialização para um Pod, que será executado antes de um 
+aplicativo ser iniciado.
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+<!-- steps -->
+
+## Crie um Pod que tem um Contêiner de incialização
+
+Neste exercício, você cria um pod que possui um aplicativo de contêiner e um
+contêiner de inicialização. O contêiner de incialização é executado até a sua conclusão, antes da aplicação
+do contêiner ser iniciada.
+
+Aqui está o arquivo de configuração do pod:
+
+{{< codenew file="pods/init-containers.yaml" >}}
+
+No arquivo de configuração, você pode ver que o Pod tem um volume 
+que o contêiner de inicialiazação e o contêiner da aplicação compartilham.
+
+O contêiner de incialização monta o volume compartilhado em `/work-dir`, e o contêiner 
+da aplicação monta o volume compartilhado em `/usr/share/nginx/html`. 
+O contêiner de incialização executa o seguinte comando e depois termina:
+
+    wget -O /work-dir/index.html http://info.cern.ch
+
+Observe que o contêiner de inicialização grava o arquivo `index.html` no diretório raiz
+do servidor nginx.
+
+Crie o Pod:
+
+    kubectl apply -f https://k8s.io/examples/pods/init-containers.yaml
+
+Verifique se o container nginx está executando:
+
+    kubectl get pod init-demo
+
+A saída mostra que o contêiner nginx está em execução:
+
+    NAME        READY     STATUS    RESTARTS   AGE
+    init-demo   1/1       Running   0          1m
+
+ Abra um shell dentro do contêiner nginx, executando o comando abaixo, no Pod init-demo:
+
+    kubectl exec -it init-demo -- /bin/bash
```

Comment:
````suggestion
```shell
kubectl exec -it init-demo -- /bin/bash
```
````

Same as the previous fenced code blocks.